[
  {
    "project": "pyccel/pyccel",
    "commit": "e01d5bf62b8c94e718c3a3ccfe6e159a827a3129",
    "filename": "tests/errors/syntax/ex1.py",
    "min_patch_found": false,
    "full_warning_msg": "tests/errors/syntax/ex1.py:1:1 Unbound name [10]: Name `a` is used but not defined in the current scope.",
    "exception": "PatchSet should only have 1 Patch, but it has 0",
    "dd_fail": true
  },
  {
    "project": "pyccel/pyccel",
    "commit": "e01d5bf62b8c94e718c3a3ccfe6e159a827a3129",
    "filename": "tests/errors/syntax/ex1.py",
    "min_patch_found": false,
    "full_warning_msg": "tests/errors/syntax/ex1.py:6:12 Unbound name [10]: Name `B` is used but not defined in the current scope.",
    "exception": "PatchSet should only have 1 Patch, but it has 0",
    "dd_fail": true
  },
  {
    "project": "pyccel/pyccel",
    "commit": "e01d5bf62b8c94e718c3a3ccfe6e159a827a3129",
    "filename": "tests/errors/syntax/ex1.py",
    "min_patch_found": false,
    "full_warning_msg": "tests/errors/syntax/ex1.py:6:15 Unbound name [10]: Name `C` is used but not defined in the current scope.",
    "exception": "PatchSet should only have 1 Patch, but it has 0",
    "dd_fail": true
  }
]